packages/backend-core/src/types/core.ts

```diff
@@ -30,7 +30,8 @@ export type TokenType = 'cookie' | 'header';
 export type AuthState = {
   status: AuthStatus;
   session?: Session;
-  interstitial?: string;
+  /* Interstitial is returned as null when the client will handle the interstitial logic */
```

Comment:
❓ What does `the client` refer-to? Are we talking about npm packages using `@clerk/backend-core`?

Reply:
This refers to the backend client code, will clarify.

Reply:
Actually this was taken from Peter's PR and no longer applies.

Reply:
Changing to:

`/* Interstitial is returned as null when the interstitial endpoint will be rewritten to instead of being rendered directly */`